crates/bevy_ecs/src/world/entity_ref.rs

```diff
@@ -3860,6 +3860,46 @@ impl<'w, 's> FilteredEntityMut<'w, 's> {
         Self { entity, access }
     }
 
+    /// Returns a copy of the `FilteredEntityMut`.
+    ///
+    /// This is unsafe because the user is now responsible to avoid aliasing violations.
+    ///
+    /// This can be useful to mutably query multiple components from a single `FilteredEntityMut`.
+    ///
+    /// ### Example Usage
+    ///
+    /// ```
+    /// # use bevy_ecs::{prelude::*, world::FilteredEntityMut};
+    /// #
+    /// # #[derive(Component)]
+    /// # struct A;
+    /// # #[derive(Component)]
+    /// # struct B;
+    /// #
+    /// # let mut world = World::new();
+    /// # world.spawn((A, B));
+    /// #
+    /// // This gives the `FilteredEntityMut` access to `&mut A` and `&mut B`.
+    /// let mut query = QueryBuilder::<FilteredEntityMut>::new(&mut world)
+    ///     .data::<(&mut A, &mut B)>()
+    ///     .build();
+    ///
+    /// let mut filtered_entity: FilteredEntityMut = query.single_mut(&mut world).unwrap();
+    /// // SAFETY: the clone won't cause aliasing violations
```

Comment:
This safety comment needs to be improved, laying out exactly why it's not aliasing.

Reply:
will update